frontend/src/tests/ddah-tests.js

```diff
@@ -65,32 +67,100 @@ export function ddahTests(api) {
             ],
         };
 
-        resp1 = await apiPOST(`/admin/ddahs`, newDdah);
-        expect(resp1).toHaveStatus("success");
-        expect(resp1.payload.duties.length).toEqual(newDdah.duties.length);
-        expect(computeTotalHoursForDdah(resp1.payload)).toEqual(
+        let resp = await apiPOST(`/admin/ddahs`, newDdah);
+        expect(resp).toHaveStatus("success");
+        expect(resp.payload.duties.length).toEqual(newDdah.duties.length);
+        expect(computeTotalHoursForDdah(resp.payload)).toEqual(
             computeTotalHoursForDdah(newDdah)
         );
 
         // We've passed the insert test, so save the DDAH for later
-        Object.assign(ddah, resp1.payload);
+        Object.assign(ddah, resp.payload);
     });
 
     it("get a ddah", async () => {
-        resp1 = await apiGET(`/admin/ddahs/${ddah.id}`);
-        expect(resp1).toHaveStatus("success");
-        expect(resp1.payload).toMatchObject(ddah);
+        let resp = await apiGET(`/admin/ddahs/${ddah.id}`);
+        expect(resp).toHaveStatus("success");
+        expect(resp.payload).toMatchObject(ddah);
     });
 
     it("get all ddahs associated with a session", async () => {
-        resp1 = await apiGET(`/admin/sessions/${session.id}/ddahs`);
-        expect(resp1).toHaveStatus("success");
-        expect(resp1.payload).toContainObject(ddah);
+        let resp = await apiGET(`/admin/sessions/${session.id}/ddahs`);
+        expect(resp).toHaveStatus("success");
+        expect(resp.payload.length).toEqual(1);
+        expect(resp.payload).toContainObject(ddah);
```

Comment:
Why are we using toEqual() on line 73 while using toContainObject() on line 91? It might be better to use the same comparison method for all ddah tests.